src/zope/configuration/fields.py

```diff
@@ -38,21 +40,33 @@
 
 @implementer(IFromUnicode)
 class PythonIdentifier(TextLine):
-    """This field describes a python identifier, i.e. a variable name.
     """
-    def fromUnicode(self, u):
-        return u.strip()
+    This field describes a python identifier, i.e. a variable name.
+
+    Empty strings are allowed.
+    """
+
+    def fromUnicode(self, value):
+        # XXX: This doesn't do validation.
```

Comment:
Given that you've opened #28 to track this issue, the comment is out-of-place.

Reply:
Good point, that predates the issue.